src/test/rustdoc-ui/feature-gate-doc_auto_cfg.rs

```diff
@@ -0,0 +1,2 @@
+// check-pass
+pub fn foo() {}
```

Comment:
This doesn't actually do anything. Can you remove this test and rename `src/test/rustdoc/doc-not-auto-cfg.rs` to be the feature gate?

Reply:
I thought that it needed to be a `ui` test. Makes things simpler if not!